tests/switchmap_/core/test_rest.py

```
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+"""Test the rest module."""
+
+import os
+import sys
+import unittest
+import requests
```

Ruff (0.8.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)